sdk/template/perf-tests/template/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@azure-tests/perf-template",
+  "sdk-type": "perf-test",
+  "version": "1.0.0",
+  "description": "Template project demonstrating use of the perf framework.",
+  "main": "",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@azure/app-configuration": "^1.3.1",
+    "@azure/core-http": "^2.0.0",
+    "@azure/test-utils-perf": "^1.0.0",
+    "dotenv": "^8.2.0",
+    "@azure/template": "^1.0.11-beta.1",
```

```suggestion
    "@azure/template": "^1.0.11-beta.1 || ^1.0.12-beta.1",
```

This should fix the build break.  You can merge this for now and @praveenkuttappan and/or I will investigate this further and determine if this is the correct long-term fix.